lib/Dialect/TTMetal/IR/TTMetalOps.cpp

```diff
@@ -89,8 +89,9 @@ ::mlir::LogicalResult DispatchOp::verify() {
   // Assert block inputs are CBs
   for (auto &region : getRegions()) {
     for (auto arg : region.getArguments()) {
-      if (not mlir::isa<ttkernel::CBType>(arg.getType())) {
-        return emitOpError("Block inputs must be CBType");
+      if (not(mlir::isa<ttkernel::CBType>(arg.getType()) ||
```

Comment:
It's not your code, but can we convert the `not` to `!` since the line was touched?